crates/pathfinder/src/bin/pathfinder/config.rs

```diff
@@ -235,6 +235,16 @@ This should only be enabled for debugging purposes as it adds substantial proces
         default_value = "100000"
     )]
     get_events_max_uncached_bloom_filters_to_load: std::num::NonZeroUsize,
+
+    #[arg(
+        long = "storage.prune-merkle-tries",
+        long_help = r"When enabled, only the last state of the Merkle tries is kept in the database. \
+            This can be used to reduce the disk space usage at the cost of not being able to generate storage proofs only for the latest block.",
```

Comment:
```suggestion
        long_help = r"When enabled, only the last state of the Merkle tries is kept in the database. \
            This can be used to reduce the disk space usage at the cost of only being able to provide storage proofs for the latest block".
```